packages/playground/website/src/components/error-report-modal/index.tsx

```diff
@@ -119,7 +119,11 @@ export function ErrorReportModal(props: { blueprint: Blueprint }) {
 				<>
 					We were unable to submit the error report. Please try again
 					or open an{' '}
-					<a href="https://github.com/WordPress/wordpress-playground/issues/">
+					<a
```

Comment:
Unrelated changes. Links without target blank can lead to users losing their sites while in temporary mode, so we should avoid these.